aidial_adapter_bedrock/dial_api/request.py

```diff
@@ -51,3 +56,39 @@ def tools_mode(self) -> ToolsMode | None:
         if self.tool_config is not None:
             return self.tool_config.tools_mode
         return None
+
+
+def collect_text_content(
+    content: str | List[MessageContentPart] | None,
+    delimiter: str = "\n",
```

Comment:
I saw some examples, where different parts of content were separated by more verbose `\n\n`

Maybe we should do the same?

p.s.: I don't have some test result to prove that's affecting model response though

Reply:
Fixed